### PR TITLE
finished ETW spike

### PR DESCRIPTION
**Pull request by analogrelay**
Super hacky, but I wanted to get a spike of using ETW in before I have to jump off to some DNX work this afternoon.

/cc @NTaylorMullen @pranavkm

### CODE REVIEW

File: src/Microsoft.AspNet.StressFramework/StressTestRunner.cs [hunk 1]
```diff
@@ -82,14 +82,23 @@ public class StressTestRunner : TestRunner<StressTestCase>
...
+                var stopwatch = new Stopwatch();
+                using (var host = LaunchHost(TestMethod))
+                {
+                    host.AttachCollectors();
+
+                    StressTestTrace.WriteLine("Releasing Host");
+                    stopwatch.Start();
+                    host.BeginIterations();
+
+                    host.ProcessCollectorsAsync();
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNet.StressFramework/StressTestRunner.cs]: Curious: Why are we not able to attach collectors and then start them immediately?

Comment (analogrelay) [inline src/Microsoft.AspNet.StressFramework/StressTestRunner.cs]: We could. I'll probably do that. Calling Process will handle any backlog of events so it's not too risky to do it like this.

**Comment by analogrelay** [comment]
Ping @NTaylorMullen @rynowak @pranavkm 

I had to overhaul this a bit after the Host/Driver model got in. I also had to tweak the Host/Driver model a bit. Unfortunately Git didn't like a lot of the changes so they are delete/add pairs :(.

This adds a simple collector that uses ETW to gather the elapsed time for each individual iteration. I also had to restrict the stress testing to dnx451 for now in order to get ETW to work. We'll need to sync up on the best plan for this tomorrow. The ETW code _should_ work on dnxcore on Windows, we just need to port it. Obviously Linux/Unix are a different matter...

**Comment by analogrelay** [comment]
I managed to abstract enough ETW away to enable us to build/run on dnxcore50. No metrics are collected though.

**Pull request closed without merging**